plugins/default/lib/openshift.py

```diff
@@ -7,7 +7,8 @@ def oc(*args):
 
 
 def _get_service_selectors(oc, project, service):
-    svc = json.loads(oc("-n", project, "get", "service", service, "-o", "json"))
+    svc = json.loads(
```

Comment:
Some of these formatting changes are due to running autopep8 against the file as per the recommendations in the README of this repo.

Example:
```
$ autopep8 plugins/default/lib/openshift.py --in-place
```